src/main/java/io/dashboardhub/pipelinedashboard/listener/AuthenticationListener.java

```diff
@@ -20,5 +21,9 @@ public void onApplicationEvent(final InteractiveAuthenticationSuccessEvent event
         if (user == null) {
             this.userService.save(new User(event.getAuthentication().getName()));
         }
+
+        user = this.userService.findByUsername(event.getAuthentication().getName());
+        user.getLogins().add(new Login());
```

Comment:
is there a benefit to having this logged in db over log file? alternatively, could we incorporate this into Envers auditing by adding a lastLoggedIn field to the User object?